controller/handler/trial.js

```diff
@@ -11,6 +11,7 @@ const processComplianceCount = require('../helper/process-compliance-count');
 const processRules = require('../helper/process-rules');
 const processPatientStatus = require('../helper/process-patient-status');
 const httpNotFound = 404;
+const sqlDateFormat = 'ddd MMM DD YYYY HH:mm:ss ZZ';
```

Comment:
"sqlDateFormat" is used to map the date format from mySQL to be used by moment.js
Later I formatted the date in "MM-DD-YYYY"

It seems code is working without it too.